assets/javascripts/kitten/components/videos/video/index.js

```diff
@@ -20,10 +55,31 @@ const StyledVideo = styled.video`
 `
 
 export class Video extends PureComponent {
-  constructor(props) {
-    super(props)
+  video = createRef()
+  state = { showPlayer: false }
 
-    this.video = createRef()
+  handlePlayClick = () => {
+    if (this.state.showPlayer && !this.props.autoPlay) {
+      this.video.current.pause()
```

Comment:
Je ne suis pas sûr de bien comprendre ces tests pour jouer ou mettre en pause la vidéo.
Si le player est affiché, et si la vidéo n'est pas en auto-play, alors on met la vidéo sur pause ?
Sinon on met la vidéo en lecture.
Ça veut dire que si on a une vidéo en auto-play, on a aucun moyen de la mettre en pause ? C'est assez barbare de forcer la main de l'utilisateur comme ça, non ?
Pourquoi on ne ferait pas plutôt  un early return si on affiche pas le player `this.state.showPlayer`.
J'ai aussi l'impression qu'il manque une info pour gérer le play et le pause.
L'autoplay seul ne suffit pas.
Si tu joues la vidéo parce qu'elle est en autoplay, comment tu gères la pause ?
Est-ce qu'il ne faudrait pas vérifier si le player est en état `pristine` (qu'il n'a jamais été touché) et qu'il est en auto-play, alors tu play la vidéo.
Ensuite, peut importe l'autoplay ou pas. Si on appuie sur `enter` ou `espace` t'as juste à vérifier l'état de la vidéo pour la lancer ou la mettre en pause.

Reply:
Ce composant `Video` est celui de la homePage qui est joué en `autoPlay` sans aucun moyen de gérer la lecture. Oui c'est barbare mais ce fonctionnement était déjà celui-ci avant que j'ajoute le button.
Ici en ajoutant le btn, ça permettra de jouer avec le `play` et `pause` de la video quand la props `autoPlay` n'est pas déclarée.

Le fonctionnement de l'`autoPlay` déjà en place doit rester tel qu'il est.